zubhub_frontend/zubhub/src/views/create_project/step1/Step1.jsx

```diff
@@ -102,7 +102,7 @@ function Step1({ formik, handleBlur }) {
           clearSuggestions={clearSuggestions}
           removeTag={removeTag}
           handleBlur={formik.handleBlur}
-          placeholder="Start typing to materials used"
+          placeholder="Start typing your materials used"
```

Comment:
this needs translation

Reply:
As previously said, translation for this case is not with respect to this issue. @tuxology will be advanceable to do the translation on this PR, I'm trying to keep it as focused as possible.